### DISCUSSION TITLE
URL for Config?

### DISCUSSION THREAD

**Discussion by efaden** [tony-fav/tasmota-blerry#38] [Ideas]
It would be interesting if you could define a URL to store the config and have blerry pull the config from there.   It could monitor for changes and automatically pull the new config.  This way you could have a bunch of Tasmota Blerry devices throughout the house and centralize the config.

**Comment by tony-fav** [comment]
We discussed this in the discord but I wanted to put the answer here for anyone interested. The solution to this problem is to push configs with the Tasmota commands `BlerrySetDevice` or `BlerrySetConfig` via some automation platform.

**Comment by vaparr** [comment]
https://github.com/tony-fav/tasmota-blerry/pull/56
